src/util/fetch.js

```diff
@@ -1,5 +1,6 @@
 const { promiseTimeout, TimeoutError } = require('./promise');
 const _ = require('lodash');
+const fetch = require('node-fetch');
```

Comment:
This duplicates the 'node-fetch' in `server/index.js`

Reply:
This is also the reason the tests are failing, because it stomps on my mock server response. Remove.

Reply:
@jvwong this change should not break anything.. there is something wrong in the app, perhaps a bigger hidden issue... I guess, we should not probably use fetch.js file name and import it as "const fetch=require('../../fetch'") where it is used.. or use another name for the const...